File: Views/src/components/services-provision/bench/dto/service-bench-dto.ts
import { CustomerDto } from "src/components/main/customer/dtos/customer-dto";
import { BenchToCashBoxDto } from "./bench-to-Cash-Box-Dto";

export class ServiceBenchDto {
   id:number;
   customerId:number;
   customer:CustomerDto;
   dateServiceStarted:Date;
   dateServiceFinished: Date;
   remote:boolean;
   remoteAccessData:string;
   visually:string; //localAccessData
   status:string;
   finished:boolean;
   listBenchToCashBox:BenchToCashBoxDto[];
}

File: Application/Dto/ServiceBudgetBench/ServiceBenchDto.cs
using System;
using System.Collections.Generic;

namespace Application.Dto.ServiceBudgetBench
{
    public class ServiceBenchDto
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public DateTime DateServiceStarted { get; set; }
        public DateTime DateServiceFinished { get; set; }
        public bool Remote { get; set; }
        public string RemoteAccessData { get; set; }
        public string Visually { get; set; } // public string LocalAccessData { get; set; }
        public string Status { get; set; }
        public bool Finished { get; set; }
        public List<BenchToCashBoxDto> ListBenchToCashBox { get; set; }
    }
}
